circle.yml

```
@@ -86,6 +86,29 @@ commands:
         working_directory: ~/build
         command: ctest -E 'unittests|smoketests' -j4 --schedule-random --output-on-failure
 
+  collect_coverage_data:
```

Other commands use hyphens for delimiters
```suggestion
    collect-coverage-data:
```

---

I'm following CircleCI convention for command names, e.g. `store_artifacts`.

---

ok, that were job names that use hyphens